rules/dropbox_rules/dropbox_linked_team_application_added.yml

```diff
@@ -0,0 +1,121 @@
+AnalysisType: rule
+Description: An application was linked to your Dropbox Account
+DisplayName: Dropbox Linked Team Application Added
+Enabled: false
```

Comment:
Is there a reason to default this to `enabled: false`? If not, let's turn it to `enabled: true`

Reply:
Good catch; that was left over from testing. Addressed in [`1297205` (#687)](https://github.com/panther-labs/panther-analysis/pull/687/commits/1297205451fb38f8c92a23625fda514f8c5bb864).